app/components/post_list/load_more_button.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent, PropTypes} from 'react';
```

Comment:
Why this have a `load_more_button`? I thought we were going to load more posts once we reached the end of the list